docs/src/pages/components/data-grid/getting-started/getting-started.md

```diff
@@ -242,9 +238,21 @@ StackOverflow is also visited from time to time by the maintainers of MUI.
 
 [Post a question](https://stackoverflow.com/questions/tagged/mui)
 
-### Enterprise support
-
-We provide a [private support channel](https://support.mui.com/hc/en-us) for enterprise customers.
+### Professional support
+
+When purchasing an MUI X Pro or Premium license you get access to professional support for a limited duration.
+Support is available on multiple channels, but the recommended channel is GitHub issues.
+If you need to share private information, you can also use email.
+
+- **MUI X Pro**: No SLA is provided but MUI's maintainers give these issues more attention than the ones from the Community plan. The channels:
+  - GitHub: [Open a new issue](https://github.com/mui-org/material-ui-x/issues/new/choose) and leave your Order ID.
+  - Email (only to share private information): [Open a new issue](https://support.mui.com/hc/en-us/requests/new?tf_360023797420=mui_x) or send an email at x@mui.com.
+- **MUI X Premium**: Same as MUI X Pro with priority over Pro and a 48 hours SLA for the first answer.
```

Comment:
```suggestion
- **MUI X Premium**: Same as MUI X Pro with priority over Pro and a 48 hours SLA for the first answer during working days.
```

Reply:
", during working hours", or ", during business hours"?

But should we define them? And if so, what are they?

Reply:
In this case, I think that we should keep it dry, and leave this to the EULA.